From aea19773118ae49523c71c3a03086652abc240c1 Mon Sep 17 00:00:00 2001
From: root <root@domU-12-31-39-04-21-8F.compute-1.internal>
Date: Sun, 22 Jul 2012 16:02:52 +0000
Subject: [PATCH] Fixes make update

---
 Makefile |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ed79aa1..862b4e6 100644
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ install: vendors-install perms build-bootstrap
 
 reinstall: remove-vendors vendors-install
 
-update: vendors-update perms build-bootstrap
+update: vendors-update build-bootstrap clear perms
 
 vendors-install:
    @echo "Instalando vendors"
@@ -28,6 +28,7 @@ vendors-update:
 
 clear:
    @echo "Limpando cache"
+   app/console cache:clear --env=prod --no-debug
    app/console cache:clear
    @echo "Feito!"
 
-- 
1.7.9.5
